Mathlib/Topology/Homotopy/HomotopyGroup.lean

```diff
@@ -350,7 +350,9 @@ def transAt (i : N) (f g : Ω^ N X x) : Ω^ N X x :=
       dsimp only [Path.trans, fromLoop, Path.coe_mk_mk, Function.comp_apply, mk_apply,
         ContinuousMap.comp_apply, toContinuousMap_apply, funSplitAt_apply,
         ContinuousMap.uncurry_apply, ContinuousMap.coe_mk, Function.uncurry_apply_pair]
-      split_ifs; change f _ = _; swap; change g _ = _
+      split_ifs
+      on_goal 1 => change f _ = _
+      on_goal 2 => change g _ = _
```

Comment:
Maybe
```suggestion
      · show f _ = _ by congr 1
      · show g _ = _ by congr 1
```
and kill the last line

Reply:
It works, and there is no need for the `by`s.